crates/test-programs/Cargo.toml

```diff
@@ -19,4 +19,4 @@ getrandom = "0.2.9"
 futures = { workspace = true, default-features = false, features = ['alloc'] }
 url = { workspace = true }
 sha2 = "0.10.2"
-base64 = "0.21.0"
+base64 = "0.21.0"
```

Comment:
Probably an editor auto-format thing: following this project's convention, we can leave this trailing new line in.

Reply:
Done. Restored the trailing new line.